cumulusci/core/utils.py

```diff
@@ -44,6 +45,31 @@ def process_bool_arg(arg):
             return False
 
 
+def process_glob_list_arg(arg):
+    """Convert a list of glob patterns or filenames into a list of files
+    The initial list can take the form of a comma-separated list, or
+    a proper list.
+
+    Note: this function processes glob patterns, but doesn't validate
+    that the files actually exist. For example, if the pattern is
+    'foo.bar' and there is no file named 'foo.bar', the literal string
+    'foo.bar' will be included in the returned files.
+    """
+    initial_list = process_list_arg(arg)
+
+    if not arg:
+        return []
+
+    files = []
+    for path in initial_list:
+        more_files = glob.glob(path, recursive=True)
+        if len(more_files):
+            files += sorted(more_files)
+        else:
+            files.append(path)
+    return files
```

Comment:
It's possible for multiple different globs to include the same file. So I think it'd be a good idea to work with `files` as a set and then convert to a list as you return it.

Reply:
hmmm. I see your point. Sets are unordered though, so we'll lose the ordering. That's not necessarily a bad thing, but it's a regression.

Reply:
If ordering is important you can use a dict, which preserve insertion order in Python 3.6+

Reply:
yeah, that's what I did.